File: BaseCode/Backend/IspWebApi/Models/In/AccountModel.cs
using Domain;
using System;


namespace IspWebApi.Models
{

    /// <summary>Represents the one of accounts for a given user</summary>
    public class AccountModel
    {
        public string UserName { get; set; }
        public string Password { get; set; }
        public string Site { get; set; }
        public string Email { get; set; }

        public Account ToEntity(Guid ownerId)
        {
            return new Account()
            {
                UserName = this.UserName,
                Password = this.Password,
                Site = this.Site,
                EmailHolder = new EmailHolder(ownerId, this.Email)
            };
        }
    }
}

File: BaseCode/Android/app/src/main/java/com/example/ispass/webapiModels/in/AccountModel.java
package com.example.ispass.webapiModels.in;

import com.google.gson.annotations.SerializedName;

public class AccountModel {

    @SerializedName("UserName")
    public String UserName;

    @SerializedName("Password")
    public String Password;

    @SerializedName("Site")
    public String Site;

    @SerializedName("Email")
    public String Email;

    public AccountModel() {
        this.Email = "";
        this.Password = "";
        this.Site = "";
        this.UserName = "";
    }
}
